website/language.py

```diff
@@ -45,7 +45,7 @@
 
 # Shown at login page if user tries to access a resource that requires auth
 MUST_LOGIN = '''
-You must log in to access this resource.
+You must log in or create a new account to claim the contributor-ship.
```

Comment:
`MUST_LOGIN` is only used for `claim_registered_user` now, so I modified the language for better context.